### PR TITLE
VP-4553: Change configuration

### PR DESCRIPTION
**Pull request by KKisilevsky**
### Proposed changes:

- Add redis service 
- Change platform init image and script for upload assets
- Add configmap for env
- Auto reloader deployment when configmap changes

### CODE REVIEW

File: webstore-app/overlays/qa/platform-deploy.yaml [hunk 1]
```diff
@@ -3,38 +3,48 @@ kind: Deployment
...
- #       - name: vc-platform-web   
- #         env:
- #           - name: VirtoCommerce__SampleDataUrl
- #             value: https://vcbacpac.blob.core.windows.net/dac-packages/webstore_exported_data_without_binary.zip
-
+              value: webstore-platform_source                
+      containers:
+        - name: vc-platform-web   
+          env:
+            - name: REDIS_PASS
```
Comment (mvktsk) [inline webstore-app/overlays/qa/platform-deploy.yaml:42]: Switch off Redis from QA environment

**Review by mvktsk — CHANGES_REQUESTED** [review]

File: webstore-app/overlays/qa/platform-deploy.yaml (hunk 1 above)
Comment (dimitri-ak) [reply webstore-app/overlays/qa/platform-deploy.yaml:42]: Why do you think we need to switch off the Redis service from QA?

File: webstore-app/overlays/qa/cms-content-cm.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: cms-content-cm
+  annotations:
+    reloader.stakater.com/match: "true"
+data:
+  VC_PLATFORM_MODULES_CONFIG: https://raw.githubusercontent.com/VirtoCommerce/vc-deploy-apps/qa/webstore-app/overlays/qa/modules.json?8439F69CD65989A8EDE4F37181261D2E9964E81DB0F94742F9FE82C4830AC883
```
Comment (mvktsk) [inline webstore-app/overlays/qa/cms-content-cm.yaml:8]: Modules is not CMS content! May be rename cms-content-cm.yaml into content-cm.yaml?

**Review by mvktsk — CHANGES_REQUESTED** [review]

File: webstore-app/overlays/qa/platform-deploy.yaml (hunk 1 above)
Comment (mvktsk) [reply webstore-app/overlays/qa/platform-deploy.yaml:42]: @dimitri-ak Redis is an independent feature and we need to test it independent from current features that are ready for testing

**Review by mvktsk — APPROVED** [review]

**Pull request merged**